beluga_ros/test/include/beluga_ros/test/point_cloud_utils.hpp

```diff
@@ -0,0 +1,436 @@
+// Copyright 2024 Ekumen, Inc.
```

Comment:
Date.

Reply:
Fixed in 4dd4e03aa2b1457381049d8d29f5c38ef18663d8.